EmailSender.Api/Controllers/MailController.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace EmailSender.Api.Controllers
 {
-    [Route("api/mail")]
+    [Route("mail")]
```

Comment:
we have agreements in our team that we should use "api" prefix, so please return it back